cloudapp/src/app/main/main.component.ts

```diff
@@ -37,6 +38,7 @@ export class MainComponent implements OnInit {
     private lastUsedOptionsService: LastUsedOptionsService,
     private printSlipReportService: PrintSlipReportService,
     private zone: NgZone,
+    private excelExportService: ExcelExportService,
```

Comment:
Nitpick. Please try to keep these in alphabetical order.

Reply:
Resolved with commit b075a3c